components/tcf/firstLayer/src/index.js

```diff
@@ -0,0 +1,101 @@
+import React, {useState, useEffect} from 'react'
+import PropTypes from 'prop-types'
+import SuiButton from '@s-ui/react-atom-button'
+import SuiModal from '@s-ui/react-molecule-modal'
+import IconClose from './components/IconClose'
+import InfoCard from './components/InfoCard'
+
+export default function TcfFirstLayer({
+  loadUserConsent,
+  getVendorList,
+  saveUserConsent,
+  openSecondLayer
+}) {
+  const [saveButtonDisabled, setSaveButtonDisabled] = useState(true)
+  const [state, setState] = useState(null)
+  const [VLState, setVLState] = useState(null)
```

Comment:
VLState means "Vendor List State"?? use understandable naming for future maintenance, maybe VendorListState is nicer :)